draft-ietf-tls-ctls.md

```diff
@@ -292,23 +185,20 @@ layer between the handshake and the record layer:
 +---------------+---------------+---------------+
 ~~~~~
 
-Specializations are defined by a "compression profile" that specifies what
-features are to be optimized out of the handshake.  In the following
-subsections, we define the structure of these profiles, and how they are used in
-compressing and decompressing handshake messages.
-
-[[OPEN ISSUE: Do we want to have an explicit cTLS extension
-indicating that cTLS is in use and which specialization is in
-use? This goes back to whether we want the use of cTLS to
-be explicit.]]
-
+By assuming that out-of-band agreements took place already prior to the start of
+the cTLS protocol exchange, the amount of data exchanged can be radically reduced. 
+Because different clients may use different compression templates and because multiple 
+compression templates may be available for use in different deployment environments, 
+a client needs to inform the server about the profile it is planning to use. The 
+profile field in the ClientHello serves this purpose. 
 
-## Specifying a Specialization
+While it is possible to apply the template-based serialization to the entire 
+TLS 1.3 protocol, this version of the specification focuses on the certificate-based
+exchange. For other exchanges, the standard TLS 1.3 protocol is used instead. Most
+exchanges in TLS 1.3 are highly optimized and do not require compression to be used. 
```

Comment:
```suggestion
Although the template-based specialization mechanisms described here are general,
we also include specific mechanism for certificate-based exchanges because those are
where the most complexity and size reduction can be obtained. Most
exchanges in TLS 1.3 are highly optimized and do not require compression to be used. 
```